src/api/Live25API.js

```diff
@@ -0,0 +1,19 @@
+import masontodayAPI from './masontodayAPI';
+
+class Live25API {
+    constructor() {
+        this.urlprefix = 'api/25live';
+    }
+
+    async getData() {
+        try {
+            const response = await fetch(`${masontodayAPI.URL}${this.urlprefix}`);
```

Comment:
I don't think it's necessary to have this `masontodayAPI` dependency. I would change this class to have a `masontodayAPIURL` property and set it in the constructor and just use that

Reply:
i'm going to leave this, i think it makes it easier to point to the overall api, as we eventually will have more wrappers for different functions of the api